src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="search-container">
+      <form
+        class="search-form form-big"
+        data-qa="big"
+        action="#"
+        method="post"
+      >
+        <input
```

Comment:
Add `name` attribute to each input and remember that each name value should be unique, also read the checklist 6

Reply:
Not fixed. Please add `name` attribute to each input tag with naming in camelCase like it shows in checklist 6